numba/cpython/hashing.py

```diff
@@ -264,7 +264,7 @@ def _PyHASH_XXROTATE(x):
         @register_jitable(locals={'x': types.uint64})
         def _PyHASH_XXROTATE(x):
             # Rotate left 13 bits
-            return ((x << types.uint64(13)) | (x >> types.uint64(16)))
+            return ((x << types.uint64(13)) | (x >> types.uint64(19)))
```

Comment:
For reference, this line https://github.com/python/cpython/blob/b738237d6792acba85b1f6e6c8993a812c7fd815/Objects/tupleobject.c#L362 is the source of these constants.